charts/clamav/templates/service.yaml

```diff
@@ -2,6 +2,12 @@ apiVersion: v1
 kind: Service
 metadata:
   name: {{ include "clamav.fullname" . }}
+{{- if .Values.service.annotationsEnabled -}}
```

Comment:
Thanks! I would prefer an empty by default annotations field, without this flag! Closer than what is done in the community.

Reply:
done